Jellyfin.Server/Program.cs

```diff
@@ -126,38 +126,41 @@ private static ServerApplicationPaths createApplicationPaths(StartupOptions opti
                         }
                     }
                     programDataPath = Path.Combine(programDataPath, "jellyfin");
-                    // Ensure the dir exists
-                    Directory.CreateDirectory(programDataPath);
                 }
             }
+            Directory.CreateDirectory(programDataPath); // Ensure the dir exists
 
             string configDir = Environment.GetEnvironmentVariable("JELLYFIN_CONFIG_DIR");
             if (string.IsNullOrEmpty(configDir))
             {
                 if (options.ContainsOption("-configdir"))
                 {
                     configDir = options.GetOption("-configdir");
+                    Directory.CreateDirectory(configDir); // Ensure folder exists
                 }
                 else
                 {
                     // Let BaseApplicationPaths set up the default value
                     configDir = null;
                 }
             }
+            Directory.CreateDirectory(configDir); // Ensure folder exists

```

Comment:
This would bug out if `configDir` is `null` which is possible when neither env nor command line set the option.

Reply:
Removed the other commit.  Hopefully this is ok now.